(apm/php) Docs updates for release 0.34.0 (#5781)
DataDog/documentation: content/en/tracing/setup/php.md

* Unhandled exceptions, including stacktraces if available
* A total count of traces (e.g., web requests) flowing through the system

> **Note**: If you are not using any autoloading mechanism (i.e. you are neither using composer nor registering any autoloader through `spl_autoload_register`) you need to set the following environment variable in order for automatic instrumentation to be triggered: `DD_TRACE_NO_AUTOLOADER=true`.

Reviewer: Just a little tweak on the wording. :)

```suggestion
**Note**: If your application does not use Composer nor an autoloader registered with `spl_autoload_register()`, set the environment variable, `DD_TRACE_NO_AUTOLOADER=true`, to enable automatic instrumentation.
```

### Environment Variable Configuration

| Env variable                              | Default     | Note                                                                                                              |
|-------------------------------------------|-------------|-------------------------------------------------------------------------------------------------------------------|
| `DD_AGENT_HOST`                           | `localhost` | The Agent host name                                                                                               |
| `DD_AUTOFINISH_SPANS`                     | `false`     | Whether spans are automatically finished when the tracer is flushed                                               |
| `DD_DISTRIBUTED_TRACING`                  | `true`      | Whether to enable distributed tracing                                                                             |
| `DD_INTEGRATIONS_DISABLED`                | `null`      | CSV list of disabled extensions; e.g., `curl,mysqli`                                                              |

> | `DD_PRIORITY_SAMPLING`                    | `true`      | Whether to enable priority sampling.                                                                              |

Reviewer 2: ```suggestion
| `DD_PRIORITY_SAMPLING`                    | `true`      | Whether to enable priority sampling                                                                              |
```
